vuepress/docs/next/docs/consume/custom-resources.md

```diff
@@ -55,7 +55,7 @@ The Entando Operator itself is also implemented as a Docker Image. You can have
 [entando-k8s-controller-coordinator](https://github.com/entando-k8s/entando-k8s-controller-coordinator) project on Github.
 Apart from the typical Maven, Java and Docker files, you will also notice the
 [entando-k8s-controller-coordinator Helm Chart](https://github.com/entando-k8s/entando-k8s-controller-coordinator/tree/master/charts/entando-k8s-controller-coordinator).
-This Helm Chart is basically the entry point for installations of Entando on Kubernetes. More detailed instructions
+This Helm Chart is the entry point for installations of Entando on Kubernetes. More detailed instructions
```

Comment:
Well, this whole page really needs to be reviewed. We're not using Helm in the install guides anymore. Maybe we can just remove lines 56-58 but leave the "More detailed instructions..." comment. Not sure how if that breaks the page. An alternative is put a post-7.0 ticket in the backlog.